src/wasm/kotlin/java/arcs/BUILD

```diff
@@ -1,5 +1,5 @@
 load("//third_party/bazel_rules/rules_kotlin/kotlin/native:native_rules.bzl", "kt_native_library")
-load("//tools/build_defs/kotlin:rules.bzl", "kt_jvm_library")
```

Comment:
Might be worth grepping the codebase for all other usages of kt_jvm_library and replace those too.

Then also clean up all of the `@file:Suppress` annotations I added everywhere, they won't be needed anymore.

And also, please run a TAP presubmit ;-)

Reply:
Running TAP now :)